### PR TITLE
Added capitalize function

### PR DESCRIPTION
**Pull request by satcasm**
Issue Reference: #440 

Earlier the first letter of the ```blogTitle``` was not necessarily capitalized but after adding this function , even if the user enters the title in lowercase it will make the first letter capital.

### CODE REVIEW

File: routes/index.router.js [hunk 1]
```diff
@@ -183,3 +183,7 @@ router.post('/compose', auth, upload.single('photo'), async (req, res) => {
 });
 
 module.exports = router;
+
+function capitalize(str) {
+  return str.charAt(0).toUpperCase() + str.slice(1);
+}
```
Comment (Pranav016) [inline routes/index.router.js:189]: I think putting at the end would make other contributors think where this was used..
Instead put it above the router it is used and label it with comments

**Review by Pranav016 — CHANGES_REQUESTED** [review]

File: routes/index.router.js (hunk 1 above)
Comment (satcasm) [reply routes/index.router.js:189]: Sure.

Comment (Pranav016) [reply routes/index.router.js]: Or you can keep it at the end, just add comment like this.
```
<- All Functions used in the file ->

<- Capitalize function used in ....... ->
function here
```

Comment (satcasm) [reply routes/index.router.js]: I have made the changes have a look and tell me if anything else needs to be changed.
Thanks!

**Pull request merged**